airtime_mvc/application/models/Preference.php

```diff
@@ -498,7 +498,12 @@ public static function SetStationDescription($description)
 
     public static function GetStationDescription()
     {
-        return self::getValue("description");
+        $description = self::getValue("description");
+        if (!empty($description)) {
+            return $description;
+        } else {
+            return "Powered by Airtime Pro";
```

Comment:
Using a placeholder is better than nothing, although then the translator might need to be provided with some context. For example 'Powered by %' in the case of Airtime Pro is an idiom and might be difficult to translate, compared to say 'Powered by % lithium-ion batteries' where % is the brand of batteries.